servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1643,6 +1667,94 @@ public static Completable mergeAllDelayError(int maxConcurrency, Completable...
+     *      }
+     * }</pre>
+     *
+     * @param completables {@link Completable}s the result of which are to be ambiguated.
+     * @return A new {@link Completable} that terminates with the result (either success or error) of whichever amongst
+     * the passed {@code completables} that terminates first.
+     * @see <a href="http://reactivex.io/documentation/operators/amb.html">ReactiveX amb operator.</a>
+     */
+    public static Completable amb(final Completable... completables) {
+        return Single.amb(stream(spliterator(completables), false)
+                .map(Completable::toSingle).collect(toList())).ignoreElement();
+    }
+
+    /**
+     * Creates a new {@link Completable} that terminates with the result (either success or error) of whichever amongst
+     * the passed {@code completables} that terminates first.
+     * <p>
+     * From a sequential programming point of view this method is roughly equivalent to the following:
+     * <pre>{@code
+     *      for (Future<T> ft: futures) { // Provided Futures (analogous to the Completables here)
+     *          // This is an approximation, this operator will pick the first result from any of the futures.
+     *          return ft.get();
+     *      }
+     * }</pre>
+     *
+     * @param completables {@link Completable}s the result of which are to be ambiguated.
+     * @return A new {@link Completable} that terminates with the result (either success or error) of whichever amongst
+     * the passed {@code completables} that terminates first.
+     * that result.
+     * @see <a href="http://reactivex.io/documentation/operators/amb.html">ReactiveX amb operator.</a>
+     */
+    public static Completable amb(final Iterable<Completable> completables) {
+        return Single.amb(stream(completables.spliterator(), false)
+                .map(Completable::toSingle).collect(toList())).ignoreElement();
+    }
+
+    /**
+     * Creates a new {@link Completable} that terminates with the result (either success or error) of whichever amongst
+     * the passed {@code completables} that terminates first.
+     * <p>
+     * From a sequential programming point of view this method is roughly equivalent to the following:
+     * <pre>{@code
+     *      for (Future<T> ft: futures) { // Provided Futures (analogous to the Completables here)
+     *          // This is an approximation, this operator will pick the first result from any of the futures.
+     *          return ft.get();
+     *      }
+     * }</pre>
+     *
+     * @param completables {@link Completable}s the result of which are to be ambiguated.
+     * @return A new {@link Completable} that terminates with the result (either success or error) of whichever amongst
+     * the passed {@code completables} that terminates first.
+     * @see <a href="http://reactivex.io/documentation/operators/amb.html">ReactiveX amb operator.</a>
+     */
+    public static Completable anyOf(final Completable... completables) {
```

Comment:
Should we mention that this is an alias of `amb` operator and provide cross-links between `amb` and `anyOf` in `@see` tags?

Reply:
I thought of doing it but decided against as it is an implementation detail that they are the same things, users do not get any value by knowing the relationship.